modules/data-entry/src/main/frontend/src/questionnaire/SubjectSelector.jsx

```diff
@@ -199,11 +197,8 @@ function UnstyledSelectParentDialog (props) {
   let initialized = parentType && childType;
 
   return(
-    <Dialog open={open} onClose={onClose} keepMounted disableBackdropClick>
-      <DialogTitle id="new-form-title">
-        Select parent {parentType?.['label']} for new {childType?.['label']}
-      </DialogTitle>
-      <DialogContent dividers className={classes.NewFormDialog}>
+    <ResponsiveDialog open={open} onClose={onClose} keepMounted title={"Select parent " + parentType?.['label'] + " for new " + childType?.['label']}>
```

Comment:
You could use the \` kind of string:

```suggestion
    <ResponsiveDialog open={open} onClose={onClose} keepMounted title={`Select parent ${parentType?.['label']} for new ${childType?.['label']}`>
```

(untested)